Makefile

```diff
@@ -58,13 +75,21 @@ really-clean: clean
 deploy:
 	@./scripts/deploy.sh ${BROKER_IMAGE}:${TAG} ${REGISTRY} ${ORG}
 
-test:
-	go test ./pkg/...
-
 cleanup-ci:
 	./scripts/broker-ci/cleanup-ci.sh
 
-ci:
+ci: fmtcheck vet lint build test ## Run CI (fmtcheck, vet, lint, build, test)
```

Comment:
@djzager skip the `make travis` more than likely I'm going to run one of the targets to see if it failed travis. i.e. lint or format may have failed, I'll just run make lint or make format fix the problem and repush.